packages/@internationalized/date/src/queries.ts

```diff
@@ -194,6 +194,7 @@ export function startOfWeek(date: DateValue, locale: string): DateValue {
 export function endOfWeek(date: ZonedDateTime, locale: string): ZonedDateTime;
 export function endOfWeek(date: CalendarDateTime, locale: string): CalendarDateTime;
 export function endOfWeek(date: CalendarDate, locale: string): CalendarDate;
+export function endOfWeek(date: DateValue, locale: string): DateValue;
```

Comment:
ah, yep, i see this is a common source of confusion
https://www.typescriptlang.org/docs/handbook/2/functions.html#function-overloads

Thanks!